qrexec/policy/utils.py

```diff
@@ -44,11 +54,11 @@ def __init__(self, path=POLICYPATH, use_legacy=True, lazy_load=False):
         self.watches = []
         self.notifier = None
 
-    def initialize_watcher(self):
-        self.watch_manager = pyinotify.WatchManager()
+    def initialize_watcher(self) -> None:
+        self.watch_manager: pyinotify.WatchManager = pyinotify.WatchManager()
```

Comment:
(here, but also as a general remark) I don't think annotation on assignment is necessary when it's exactly the type of what's being assigned (and it isn't somehow obfuscated). Especially when the variable you assign to is already annotated. In fact, it might be even harmful, as it might hide mismatch between such annotations.